source/common/upstream/upstream_impl.h

```diff
@@ -89,7 +92,20 @@ class HostDescriptionImpl : virtual public HostDescription {
             : Network::Utility::getAddressWithPort(*dest_address, health_check_config.port_value());
   }
 
-  // Upstream::HostDescription
+  Network::TransportSocketFactory&
```

Comment:
this two method (constructor and resolveTransportSocketFactory) are long enough, can you move it to .cc?

Reply:
done. moved.